src/components/pages/LoginPage/styles.tsx

```diff
@@ -0,0 +1,262 @@
+import styled, { css } from "styled-components";
+import { Link } from "react-router-dom";
+
+import { InputStyleProps } from "./interface";
+import Wrapper from "../../atoms/Wrapper";
+import inputStyles from "../../atoms/InputStyles";
+import LeftBottomBubbles from "../../../assets/img/left-bottom-bubbles.png";
+import LeftTopBubbles from "../../../assets/img/left-top-bubbles.png";
+import RightBubbles from "../../../assets/img/right-bubbles.png";
+import { ReactComponent as Alert } from "../../../assets/img/icons/alert-triangle.svg";
+import { ReactComponent as EyeOff } from "../../../assets/img/icons/eye-off.svg";
+import { ReactComponent as Eye } from "../../../assets/img/icons/eye.svg";
+
+import {
+  white,
+  blue100,
+  blue300,
+  primaryBackground,
+  primaryText,
+  grey100,
+  grey200,
```

Comment:
Grey-ish colors needs to be checked. One of them has smaller number and in fact is darker than the other 🤓